.github/workflows/release.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: Gem Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    name: Release gem
+    runs-on: ubuntu-24.04
+    # Optional but recommended to use a specific environment
+    environment: release
+    # Prevent releases from forked repositories
+    if: github.repository_owner == 'OpenVoxProject'
+
+    permissions:
+      id-token: write
+      contents: write
+      packages: write
```

Comment:
This is copy-pasta… not sure what is required here for this.

Reply:
Only contents is required. Packages is used when publishing to any of the github package registries, ID-token for authentication with rubygems.org.

Reply:
Awesome! I updated the PR accordingly